File: tests/fixtures/rust-syntax-family/host/crates/syntax-core/src/standards.rs
pub fn parse_invoice() -> Result<u8, &'static str> {
    Ok(7)
}

pub fn handled_parse() -> u8 {
    match parse_invoice() {
        Ok(value) if value > 0 => value,
        Ok(_) => 0,
        Err(_) => 1,
    }
}

pub fn unhandled_parse() {
    let _ = parse_invoice();
}

pub const CALL_SHAPED_STRING: &str = "parse_invoice()";

File: tests/fixtures/php-syntax-family/host/src/Standards.php
<?php

declare(strict_types=1);

function parseInvoice(): string
{
    return 'parsed';
}

function handledParse(): string
{
    try {
        return parseInvoice();
    } catch (RuntimeException) {
        return 'fallback';
    }
}

function unhandledParse(): string
{
    return parseInvoice();
}

$callStringDecoy = 'parseInvoice()';
